deployments/helm-chart/templates/controller-service.yaml

```diff
@@ -20,6 +20,9 @@ spec:
   {{- end }}
 {{- end }}
 {{- if eq .Values.controller.service.type "LoadBalancer" }}
+  {{- if hasKey .Values.controller.service "allocateLoadBalancerNodePorts" }}
+  allocateLoadBalancerNodePorts: {{ .Values.controller.service.allocateLoadBalancerNodePorts }}
+  {{- end }}
```

Comment:
I think `allocateLoadBalancerNodePorts` is still in beta, I would wait until it reaches GA before adding it

Reply:
I am using 1.22 and would like to use this feature. With the chart as-is, there is no way for me to achieve this.